src-web/tableDefinitions/hcm-compliances.js

```diff
@@ -409,6 +409,7 @@ export function getDecisionList(policy, locale) {
         <span key={`${status}-status-list`} className={`status-list status-list__${status}`}>
           <LabelGroup
             collapsedText='${remaining} more'
+            expandText='Show Fewer'
```

Comment:
This key should be `expandedText`. And I suspect we'll need it sentenced cased.

Reply:
The default was `'Show Less'`, so I assumed the same casing, but can change it. The key is fixed though 😅

Reply:
Yeah--I thought I had changed it, but it looks like that commit didn't make it in. You're covering for me. 🙂

Reply:
Yes, please!